index.md

```diff
@@ -14,13 +14,14 @@ hero:
   #     link: /overview/
 
 features:
-  - title: All opcodes
-    details: A small encyclopedia of all current opcodes.
+  - title: Introduction
```

Comment:
- I want to keep the opcode link as the first one.
- I don't think it's only introductory material in there!

Reply:
done and fair point!